apepay/daemon.py

```diff
@@ -1,8 +1,10 @@
 import asyncio
```

Comment:
maybe we should move this to like an `examples` folder or something and/or replace with utils c-net can use

Reply:
yeah I think moving this back to the examples folder, and just having some of the useful bits in a submodule that can be used by the example would be the way to go